src/commands/session.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+import { FileSystem as fs } from 'zos-lib'
+import { Logger } from 'zos-lib'
+
+const signature = 'session <network>'
+const description = 'by providing <network>, commands like create, ' +
+                    'freeze, push, status and upgrade will use <network> ' +
+                    'unless overriden.'
+
+module.exports = {
+  signature, description,
+  register: function(program) {
+    program
+      .command(signature, {noHelp: true})
+      .usage('<network>')
+      .description(description)
+      .action(async function (networkName, options) {
+        fs.write('/tmp/.zos_session', networkName)
+        let log = new Logger('commands/session')
+        log.info('Using "' + networkName + '" as default network unless overriden.')
```

Comment:
we should move this logic to a script file within the `scripts` dir like the other commands

Reply:
sounds good. Quick question for coding style: Some files are with- and some are not with semicolons. What do you prefer?

Reply:
Two things here:
1. we haven't decided whether it's a must or not using semicolons
2. we have decided to use backtick string interpolation